### DISCUSSION TITLE
Hostname queue status

### DISCUSSION THREAD

**Discussion by debow77** [sidekiq/sidekiq#6197] [Q&A]
Is there a way to determine if a hostname queue is in `Quiet` mode or any other status details about the queue.

**Comment by mperham** [comment]
Do you mean paused, like this?

https://github.com/sidekiq/sidekiq/wiki/Pro-API#pausing-queues

Queues are paused. Worker processes are quieted with the TSTP signal.

**Reply by debow77** [reply]
The worker process. I just realized our code to get existing hostname queues can already detect if the process is quieted. That will allow us to exclude those queues from having new jobs sent to them.



![image](https://github.com/sidekiq/sidekiq/assets/8048889/9c17e0c3-79e6-4c3d-b3d3-8fb4c32db1e1)

**Reply by mperham** [reply]
https://github.com/sidekiq/sidekiq/wiki/API#processes
